doc/advanced.rst

```diff
@@ -159,6 +159,28 @@ Then, these modules and packages can be imported as a regular package, i.e.,
         from benchmark_utils.helper_module.submodule1 import func2
 
 
+
+.. _precompilation:
+
+Caching pre-compilation and warmup effects
+------------------------------------------
+
+For some solvers, such as solver relying on just-in-time compilation with ``numba``, the first iteration might get a longer time due to "warmup" effects. To avoid having such effect in the benchmark results, it is usually advised to call the solver once before running the benchmark, in the ``Solver.set_objective`` method. For solvers with ``stopping_strategy in {'tolerance',  'iteration'}``, simply calling the ``Solver.run`` with a simple enough value is usually enough. For solvers with ``stopping_strategy = 'callback'``, it is possible to call ``Solver.run_once``, which will call the ``run`` method with a simple callback that do not compute the objective value and stop after one iteration.
```

Comment:
```suggestion
For some solvers, such as solver relying on just-in-time compilation with ``numba``, the first iteration might get a longer time due to "warmup" effects. To avoid having such effect in the benchmark results, it is usually advised to call the solver once before running the benchmark, in the ``Solver.set_objective`` method. For solvers with ``stopping_strategy in {'tolerance',  'iteration'}``, simply calling the ``Solver.run`` with a simple enough value is usually enough. For solvers with ``stopping_strategy = 'callback'``, it is possible to call ``Solver.run_once``, which will call the ``run`` method with a simple callback that does not compute the objective value and stops at the first call to callback.
```